examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -143,6 +150,10 @@ int seq_run(const traccc::opts::detector& detector_opts,
         traccc::host::combinatorial_kalman_filter_algorithm;
     using device_finding_algorithm =
         traccc::cuda::finding_algorithm<stepper_type, device_navigator_type>;
+    using host_ambiguity_resolution_algorithm =
+        traccc::host::greedy_ambiguity_resolution_algorithm;
+    using device_ambiguity_resolution_algorithm =
+        traccc::cuda::greedy_ambiguity_resolution_algorithm;
```

Comment:
These typedefs are probably a bit overkill. The algorithms are not templated, so we might as well use the names directly.

You could argue that this setup would make experimentation with different resolution algorithms earlier in the future, but since on line 168 then you use the typename directly anyway, not sure that that's a very strong argument. :stuck_out_tongue: